Makefile

```diff
@@ -34,13 +63,21 @@ bats:
 test: test-alpine test-debian test-jdk11 test-jdk11-alpine
 
 test-alpine: bats
+	cp -f jenkins-agent 8/alpine/
 	@FOLDER="8/alpine" bats-core/bin/bats tests/tests.bats
+	rm -f 8/alpine/jenkins-agent
 
 test-debian: bats
+	cp -f jenkins-agent 8/debian/
```

Comment:
this seems to conflict with the dockerfile changes? why copying this if you reading it from 2 levels up

Reply:
The test relies on the file being in that location while the image build process no longer requires the file in that location.

I didn't have enough time to transition the tests so I made the smallest change that I could in order to allow the tests to pass.  If required, I can certainly look inside the tests to use the jenkins-agent file from its upper level location.

Reply:
sure not a blocker